src/test/java/duke/task/DeadlineTest.java

```diff
@@ -0,0 +1,33 @@
+package duke.task;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class DeadlineTest {
+
+    @Test
+    void getAdditionalInfo() {
+        Deadline d = new Deadline("deadline", "25 Jul 2021 04:30 PM");
+        assertEquals("25 Jul 2021 04:30 PM", d.getAdditionalInfo());
+    }
+
+    @Test
+    void getTag() {
+        Deadline d = new Deadline("a", "b");
+        assertEquals("D", d.getTag());
+    }
+
+    @Test
+    void testToString() {
```

Comment:
Maybe consider more consistent naming between test methods? Such as removing "test" from the names of tests.